src/service/service.py

```
@@ -197,6 +198,34 @@ async def _handle_input(
+        balance = await credit_service.get_balance(user_id, token=auth_token)
+        input["credit_state"] = {
+            "initial_balance": balance.total_available,
+            "current_balance": balance.total_available,
+            "total_used": 0,
+            "used_per_feature": {},
+            "difficulty": "easy",
+            "insufficient": False,
+            "stopped_at_feature": None,
+        }
+        logger.info(f"_handle_input: credit_balance={balance.total_available}")
+    except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)